spec/lcm/integration/support/project_helper.rb

```diff
@@ -106,6 +106,28 @@ def deploy_processes(ads)
         }
         project.deploy_process component_data
 
+        component_data = {
+          name: ADD_COMPONENT_NAME,
+          type: :etl,
+          component: {
+            name: 'gdc-data-distribution',
+            version: '1',
+            config: {
+              dataDistribution: {
+                dataSource: 'blabla'
+              }
+            }
+          }
+        }
+
+        add_component = GoodData::Process.deploy_component(
+          component_data,
+          project: project,
+          client: client
+        )
+
+        add_component.create_schedule('0 0 1 1 *', 'add-component-schedule')
```

Comment:
we should either create the schedule as disabled or manual, to prevent unnecessary runs on projects that were left after test runs on staging servers